Turn on tighter controls for mypy

Smile

```
eggbot/discordclient.py:21: error: Function is missing a return type annotation
eggbot/utils/loadenv.py:37: error: Function is missing a return type annotation
eggbot/utils/loadenv.py:37: note: Use "-> None" if function does not return a value
eggbot/eventsubs.py:33: error: Function is missing a return type annotation
eggbot/eventsubs.py:33: note: Use "-> None" if function does not return a value
eggbot/utils/configio.py:20: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:22: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:41: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:33: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:36: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/eggbotcore.py:52: error: Function is missing a type annotation for one or more arguments
eggbot/eggbotcore.py:65: error: Function is missing a type annotation for one or more arguments
tests/test_eggbotcore.py:37: error: Incompatible types in assignment (expression has type "None", variable has type "ConfigFile")
tests/test_eggbotcore.py:52: error: Incompatible types in assignment (expression has type "None", variable has type "LoadEnv")
Found 12 errors in 7 files (checked 4 source files)
eggbot/discordclient.py:21: error: Function is missing a return type annotation
eggbot/utils/configio.py:20: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:22: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:41: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:33: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:36: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
modules/shoulderbirdconfig.py:31: error: Function is missing a type annotation for one or more arguments
modules/shoulderbirdconfig.py:38: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
modules/shoulderbirdconfig.py:63: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
modules/shoulderbirdconfig.py:107: error: Function is missing a type annotation for one or more arguments
Found 10 errors in 4 files (checked 4 source files)
eggbot/utils/configio.py:20: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:22: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:41: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/eventsubs.py:33: error: Function is missing a return type annotation
eggbot/eventsubs.py:33: note: Use "-> None" if function does not return a value
eggbot/configfile.py:33: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:36: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
tests/test_eventsubs.py:22: error: Argument 2 to "add" of "EventSubs" has incompatible type "Callable[[], None]"; expected "Callable[[Any], Coroutine[Any, Any, None]]"
tests/test_eventsubs.py:23: error: Argument 2 to "add" of "EventSubs" has incompatible type "Callable[[str, Dict[Any, Any]], bool]"; expected "Callable[[Any], Coroutine[Any, Any, None]]"
tests/test_eventsubs.py:24: error: Argument 2 to "add" of "EventSubs" has incompatible type "Callable[[str, Dict[Any, Any]], bool]"; expected "Callable[[Any], Coroutine[Any, Any, None]]"
tests/test_eventsubs.py:50: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
tests/test_eventsubs.py:56: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
tests/test_module_chatkudos.py:30: error: Missing type parameters for generic type "Generator"
Found 12 errors in 5 files (checked 4 source files)
eggbot/discordclient.py:21: error: Function is missing a return type annotation
eggbot/utils/configio.py:20: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:22: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:41: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/loadenv.py:37: error: Function is missing a return type annotation
eggbot/utils/loadenv.py:37: note: Use "-> None" if function does not return a value
eggbot/eventsubs.py:33: error: Function is missing a return type annotation
eggbot/eventsubs.py:33: note: Use "-> None" if function does not return a value
eggbot/configfile.py:33: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:36: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/eggbotcore.py:52: error: Function is missing a type annotation for one or more arguments
eggbot/eggbotcore.py:65: error: Function is missing a type annotation for one or more arguments
modules/shoulderbirdconfig.py:31: error: Function is missing a type annotation for one or more arguments
modules/shoulderbirdconfig.py:38: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
modules/shoulderbirdconfig.py:63: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
modules/shoulderbirdconfig.py:107: error: Function is missing a type annotation for one or more arguments
modules/module_shoulderbirdparser.py:44: error: Function is missing a return type annotation
modules/module_shoulderbirdparser.py:44: note: Use "-> None" if function does not return a value
tests/test_module_shoulderbirdparser.py:32: error: Function is missing a return type annotation
tests/test_module_shoulderbirdparser.py:40: error: Function is missing a return type annotation
Found 17 errors in 9 files (checked 4 source files)
eggbot/discordclient.py:21: error: Function is missing a return type annotation
eggbot/eventsubs.py:33: error: Function is missing a return type annotation
eggbot/eventsubs.py:33: note: Use "-> None" if function does not return a value
eggbot/utils/configio.py:20: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:22: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:41: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/loadenv.py:37: error: Function is missing a return type annotation
eggbot/utils/loadenv.py:37: note: Use "-> None" if function does not return a value
modules/module_echobox.py:149: error: Function is missing a type annotation for one or more arguments
eggbot/configfile.py:33: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:36: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/eggbotcore.py:52: error: Function is missing a type annotation for one or more arguments
eggbot/eggbotcore.py:65: error: Function is missing a type annotation for one or more arguments
Found 11 errors in 7 files (checked 4 source files)
eggbot/discordclient.py:21: error: Function is missing a return type annotation
eggbot/utils/configio.py:20: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:22: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:41: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:33: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:36: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
tests/test_discordclient.py:24: error: "run" of "DiscordClient" does not return a value
modules/shoulderbirdconfig.py:31: error: Function is missing a type annotation for one or more arguments
modules/shoulderbirdconfig.py:38: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
modules/shoulderbirdconfig.py:63: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
modules/shoulderbirdconfig.py:107: error: Function is missing a type annotation for one or more arguments
Found 11 errors in 5 files (checked 4 source files)
eggbot/utils/configio.py:20: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:22: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:41: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/loadenv.py:37: error: Function is missing a return type annotation
eggbot/utils/loadenv.py:37: note: Use "-> None" if function does not return a value
eggbot/configfile.py:33: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:36: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
tests/test_configfile.py:41: error: Argument 1 to "create" of "ConfigFile" has incompatible type "int"; expected "str"
Found 7 errors in 4 files (checked 4 source files)
eggbot/discordclient.py:21: error: Function is missing a return type annotation
eggbot/utils/configio.py:20: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:22: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/utils/configio.py:41: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:33: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
eggbot/configfile.py:36: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
modules/shoulderbirdconfig.py:31: error: Function is missing a type annotation for one or more arguments
modules/shoulderbirdconfig.py:38: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
modules/shoulderbirdconfig.py:63: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters
modules/shoulderbirdconfig.py:107: error: Function is missing a type annotation for one or more arguments
modules/module_shoulderbirdparser.py:44: error: Function is missing a return type annotation
modules/module_shoulderbirdparser.py:44: note: Use "-> None" if function does not return a value
Found 11 errors in 5 files (checked 2 source files)
```

--- a/eggbot/configfile.py
+++ b/eggbot/configfile.py
@@ -8,13 +8,14 @@ Git Repo: https://github.com/Preocts/Egg_Bot
 """
 import logging
 from typing import Any
+from typing import Dict
 from typing import Optional
 
 from eggbot.utils.configio import ConfigIO
 
 
 class ConfigFile:
-    """ Core configuration handler """
+    """Core configuration handler"""
 
     configClient: ConfigIO = ConfigIO()
     logger = logging.getLogger(__name__)
@@ -29,36 +30,36 @@ class ConfigFile:
             filename : Path and name of the JSON config file
         """
         self.filename: Optional[str] = filename
-        self.__config: dict = {}
+        self.__config: Dict[str, Any] = {}
 
     @property
-    def config(self) -> dict:
-        """ Return copy of configuration dictionary """
+    def config(self) -> Dict[str, Any]:
+        """Return copy of configuration dictionary"""
         return dict(self.__config)
 
     def unload(self) -> None:
-        """ Unloads config without saving """
+        """Unloads config without saving"""
         self.__config = {}
 
     def load(self, filename: Optional[str] = None) -> bool:
-        """ Load config. Uses prior loaded file if none provided """
+        """Load config. Uses prior loaded file if none provided"""
         if filename:
             self.filename = filename
         self.__config = self.configClient.load(self.filename)
         return bool(self.__config)
 
     def save(self, filename: Optional[str] = None) -> bool:
-        """ Save config. Uses prior loaded file if none provided """
+        """Save config. Uses prior loaded file if none provided"""
         if filename:
             self.filename = filename
         return self.configClient.save(self.__config, self.filename)
 
     def read(self, key: str) -> Any:
-        """ Reads values by key from config. Returns None if not exists """
+        """Reads values by key from config. Returns None if not exists"""
         return self.__config.get(key)
 
     def create(self, key: str, value: Any = None) -> bool:
-        """ Creates a key/value pair in the configuration """
+        """Creates a key/value pair in the configuration"""
         if not isinstance(key, str):
             self.logger.error(".create() key not string. Given a %s", str(type(key)))
             return False
@@ -69,7 +70,7 @@ class ConfigFile:
         return True
 
     def update(self, key: str, value: Any = None) -> bool:
-        """ Updates key/value pair in the configuration """
+        """Updates key/value pair in the configuration"""
         if key not in self.__config.keys():
             self.logger.error(".update() key not found, use .create()")
             return False
@@ -77,7 +78,7 @@ class ConfigFile:
         return True
 
     def delete(self, key: str) -> bool:
-        """ Deletes key from configuration """
+        """Deletes key from configuration"""
         if key not in self.__config.keys():
             self.logger.error(".delete() key not found.")
             return False

--- a/eggbot/discordclient.py
+++ b/eggbot/discordclient.py
@@ -8,6 +8,7 @@ Git Repo: https://github.com/Preocts/Egg_Bot
 """
 from __future__ import annotations
 
+from typing import Any
 from typing import List
 from typing import Optional
 
@@ -15,44 +16,44 @@ import discord  # type: ignore
 
 
 class SingleClient(type):
-    """ Python singleton factory """
+    """Python singleton factory"""
 
     _instances: Optional[discord.Client] = None
 
-    def __call__(cls):
-        """ Provide existing instance or create one """
+    def __call__(cls) -> Any:  # type: ignore
+        """Provide existing instance or create one"""
         if not cls._instances:
             cls._instances = super(SingleClient, cls).__call__()
         return cls._instances
 
 
 class DiscordClient(metaclass=SingleClient):
-    """ Abstract layer for discord client of choice """
+    """Abstract layer for discord client of choice"""
 
     def __init__(self) -> None:
-        """ Creates a singleon of a Discord Client """
+        """Creates a singleon of a Discord Client"""
         self.__discord_secret = ""
         intents = discord.Intents(messages=True, guilds=True, members=True)
         self.client = discord.Client(status="online", intents=intents)
 
     def set_secret(self, discord_secret: str) -> None:
-        """ Store Discord API key """
+        """Store Discord API key"""
         self.__discord_secret = discord_secret
 
     def run(self) -> None:
-        """ Start discord connection - blocking """
+        """Start discord connection - blocking"""
         self.client.run(self.__discord_secret)
 
     async def close(self) -> None:
-        """ Shut down the connection """
+        """Shut down the connection"""
         await self.client.close()
 
     @property
     def guilds(self) -> List[discord.Guild]:
-        """ List of guilds loaded to client """
+        """List of guilds loaded to client"""
         return self.client.guilds
 
     @property
     def users(self) -> List[discord.User]:
-        """ List of users loaded to client """
+        """List of users loaded to client"""
         return self.client.users

--- a/eggbot/eggbotcore.py
+++ b/eggbot/eggbotcore.py
@@ -12,6 +12,9 @@ import os
 import sys
 from typing import List
 
+from discord import Member
+from discord import Message
+
 from eggbot.configfile import ConfigFile
 from eggbot.discordclient import DiscordClient
 from eggbot.eventsubs import EventSubs
@@ -20,7 +23,7 @@ from eggbot.utils.loadenv import LoadEnv
 
 
 class EggBotCore:
-    """ Main construct of bot """
+    """Main construct of bot"""
 
     DEFAULT_CONFIG = "configs/eggbotcore.json"
     MODULE_PATH = "./modules"
@@ -29,14 +32,14 @@ class EggBotCore:
     discord_ = DiscordClient()
 
     def __init__(self) -> None:
-        """ Declare class objects """
+        """Declare class objects"""
         self.event_subs = EventSubs()
         self.core_config = ConfigFile()
         self.env_vars = LoadEnv()
         self.loaded_modules: List[object] = []
 
     def load_config(self) -> bool:
-        """ Load configuration """
+        """Load configuration"""
         self.logger.info("Opening configuration...")
         if not self.core_config:
             raise Exception("Internal Object 'core_config' not initialized.")
@@ -49,8 +52,8 @@ class EggBotCore:
         )
         return True
 
-    async def on_member_join(self, member) -> bool:
-        """ Triggered on all join events """
+    async def on_member_join(self, member: Member) -> bool:
+        """Triggered on all join events"""
         if member.id == self.discord_.client.user.id:
             self.logger.warning("on_member_join(), Saw ourselves join, that's weird.")
             return False
@@ -62,8 +65,8 @@ class EggBotCore:
                 await subbed(member)
         return True
 
-    async def on_message(self, message) -> bool:
-        """ Triggered on all message events """
+    async def on_message(self, message: Message) -> bool:
+        """Triggered on all message events"""
         if message.author.id == self.discord_.client.user.id:
             self.logger.debug("on_message(), Ignoring ourselves.")
             return False
@@ -80,21 +83,21 @@ class EggBotCore:
         return True
 
     async def on_ready(self) -> bool:
-        """ Triggered when client has completed processing of data recieved """
+        """Triggered when client has completed processing of data recieved"""
         if self.event_subs:
             for subbed in self.event_subs.get(EventType.ON_READY):
                 await subbed(None)
         return True
 
     async def on_disconnect(self) -> bool:
-        """ Triggered on disconnect. Does not indicate re-connect logic will fail """
+        """Triggered on disconnect. Does not indicate re-connect logic will fail"""
         if self.event_subs:
             for subbed in self.event_subs.get(EventType.ON_DISCONNECT):
                 await subbed(None)
         return True
 
     def launch_bot(self) -> int:
-        """ Start the bot, blocking """
+        """Start the bot, blocking"""
         self.__load_environment()
         self.__load_modules()
         self.__register_events()
@@ -102,7 +105,7 @@ class EggBotCore:
         return 0
 
     def __load_environment(self) -> None:
-        """ Load core config and environment variables """
+        """Load core config and environment variables"""
         if not self.env_vars:
             raise Exception("Internal Object 'env_vars' not initialized.")
         if not self.load_config():
@@ -111,7 +114,7 @@ class EggBotCore:
         self.discord_.set_secret(self.env_vars.get("DISCORD_SECRET"))
 
     def __register_events(self) -> None:
-        """ Link event handler methods to discord client """
+        """Link event handler methods to discord client"""
         # This replaces the @discord.client.event decorators as we want to
         # capture the instance method of these, not the unbound function.
         self.discord_.client.event(self.on_ready)
@@ -120,7 +123,7 @@ class EggBotCore:
         self.discord_.client.event(self.on_message)
 
     def __load_modules(self) -> None:
-        """ Load the modules for the bot """
+        """Load the modules for the bot"""
         for module_name in self.__get_module_files():
             if module_name in sys.modules:
                 continue
@@ -131,7 +134,7 @@ class EggBotCore:
                 self.logger.error("Module not loaded: '%s' (%s)", module_name, err)
 
     def __get_module_files(self) -> List[str]:
-        """ Returns list of ./modules/module*.py files """
+        """Returns list of ./modules/module*.py files"""
         file_list: List[str] = []
         for result in os.listdir(self.MODULE_PATH):
             if result.startswith("module_") and result.endswith(".py"):
@@ -139,7 +142,7 @@ class EggBotCore:
         return file_list
 
     def __register_module_events(self, module: object, module_name: str) -> None:
-        """ Initializes module class and registers identified event subscriptions """
+        """Initializes module class and registers identified event subscriptions"""
         class_name = getattr(module, "AUTO_LOAD", None)
         if class_name is None:
             raise ModuleNotFoundError("Unable to find expected AUTO_LOAD attr")

--- a/eggbot/eventsubs.py
+++ b/eggbot/eventsubs.py
@@ -19,26 +19,26 @@ EVENTCALLBACK = Callable[[Any], Coroutine[Any, Any, None]]
 
 
 class EventAction(NamedTuple):
-    """ Event Action Object """
+    """Event Action Object"""
 
     event: EventType
     target: EVENTCALLBACK
 
 
 class EventSubs:
-    """ Hold all subscriptions to events from bot addon modules """
+    """Hold all subscriptions to events from bot addon modules"""
 
     logger = logging.getLogger(__name__)
 
-    def __init__(self):
+    def __init__(self) -> None:
         self.__pubsub: List[EventAction] = []
 
     def add(self, event: EventType, target: EVENTCALLBACK) -> None:
-        """ Add new event to event subscriptions """
+        """Add new event to event subscriptions"""
         self.__pubsub.append(EventAction(event, target))
 
     def get(self, event: EventType) -> List[EVENTCALLBACK]:
-        """ Returns a list of callable targets by event type, can be empty """
+        """Returns a list of callable targets by event type, can be empty"""
         result: List[EVENTCALLBACK] = []
         for sub in self.__pubsub:
             if sub.event == event:

--- a/eggbot/utils/configio.py
+++ b/eggbot/utils/configio.py
@@ -8,17 +8,19 @@ Git Repo: https://github.com/Preocts/Egg_Bot
 """
 import json
 import logging
+from typing import Any
+from typing import Dict
 from typing import Optional
 
 
 class ConfigIO:
-    """ Input and output abstract layer for JSON configs """
+    """Input and output abstract layer for JSON configs"""
 
     logger = logging.getLogger(__name__)
 
-    def load(self, filename: Optional[str] = None) -> dict:
-        """ Loads a config json, dumps loaded config with no prompt """
-        loaded_config: dict = {}
+    def load(self, filename: Optional[str] = None) -> Dict[str, Any]:
+        """Loads a config json, dumps loaded config with no prompt"""
+        loaded_config: Dict[str, Any] = {}
         if not filename:
             self.logger.error("No filename provided to load, aborting.")
             return loaded_config
@@ -37,8 +39,8 @@ class ConfigIO:
             self.logger.error("", exc_info=True)
         return loaded_config
 
-    def save(self, config: dict, filename: Optional[str] = None) -> bool:
-        """ Saves a config json, overwrites existing file with no prompt """
+    def save(self, config: Dict[str, Any], filename: Optional[str] = None) -> bool:
+        """Saves a config json, overwrites existing file with no prompt"""
         if not filename:
             self.logger.error("No filename provided to save, aborting.")
             return False

--- a/eggbot/utils/loadenv.py
+++ b/eggbot/utils/loadenv.py
@@ -19,28 +19,28 @@ from typing import Optional
 
 
 class LoadEnv:
-    """ Load a local .env file into environment """
+    """Load a local .env file into environment"""
 
     logger = logging.getLogger(__name__)
 
     @staticmethod
     def get(key: str) -> str:
-        """ Get a value from environ """
+        """Get a value from environ"""
         return os.environ.get(key, "")
 
     def __init__(self) -> None:
-        """ Provide a path to .env if not located in working directory """
+        """Provide a path to .env if not located in working directory"""
         self.filepath: Optional[str] = None
         self.env_values: Dict[str, str] = {}
 
-    def __del__(self):
-        """ Destructor """
+    def __del__(self) -> None:
+        """Destructor"""
         if self.env_values:
             for key in self.env_values:
                 del os.environ[key]
 
     def load(self, filepath: Optional[str] = None) -> None:
-        """ Loads local .env or from path if provided """
+        """Loads local .env or from path if provided"""
         path = filepath if filepath else "./"
         if not os.path.isfile(os.path.join(path, ".env")):
             return
@@ -51,7 +51,7 @@ class LoadEnv:
             self.__load_to_environ()
 
     def __parse_env(self, input_file: str) -> None:
-        """ Parses env file into key-pair values """
+        """Parses env file into key-pair values"""
         for line in input_file.split("\n"):
             if not line or line.strip().startswith("#") or len(line.split("=", 1)) != 2:
                 # Blank line | Comment line | Unhealthy line
@@ -60,6 +60,6 @@ class LoadEnv:
             self.env_values[key.strip()] = value.strip()
 
     def __load_to_environ(self) -> None:
-        """ Loads values to local environ """
+        """Loads values to local environ"""
         for key, value in self.env_values.items():
             os.environ[key] = value

--- a/modules/module_echobox.py
+++ b/modules/module_echobox.py
@@ -24,14 +24,14 @@ AUTO_LOAD: str = "EchoBox"
 
 
 class ReturnMessage(NamedTuple):
-    """ Data model for return values of parse command """
+    """Data model for return values of parse command"""
 
     channel: str = ""
     owner: str = ""
 
 
 class EchoBox:
-    """ Talk to a chennel through the bot """
+    """Talk to a chennel through the bot"""
 
     MODULE_NAME: str = "EchoBox"
     MODULE_VERSION: str = "1.0.0"
@@ -43,7 +43,7 @@ class EchoBox:
     logger = logging.getLogger(__name__)
 
     def __init__(self) -> None:
-        """ Create instance and load configuration file """
+        """Create instance and load configuration file"""
         self.logger.info("Initializing EchoBox module")
 
         self.discord = DiscordClient()
@@ -58,7 +58,7 @@ class EchoBox:
             self.logger.warning("BOT_OWNER not set in '.env'. EchoBox disabled.")
 
     def __populate_owner(self) -> None:
-        """ Looks up owner by ID, returns None if not found """
+        """Looks up owner by ID, returns None if not found"""
         if self.owner is not None or not self.owner_id:
             return
 
@@ -68,7 +68,7 @@ class EchoBox:
             self.owner_id = ""
 
     async def __send_to_owner(self, content: str) -> None:
-        """ Sends a DM to the provided discord.User """
+        """Sends a DM to the provided discord.User"""
         if not self.owner or not content:
             return
 
@@ -86,7 +86,7 @@ class EchoBox:
             self.logger.info("DM sent.")
 
     async def __send_to_channel(self, content: str) -> None:
-        """ Send a message to the provided channel """
+        """Send a message to the provided channel"""
         if not self.target_channel or not content:
             return
 
@@ -95,7 +95,7 @@ class EchoBox:
         await self.target_channel.send(content)
 
     def parse_command(self, message: Message) -> ReturnMessage:
-        """ Runs commands, returns messages to send """
+        """Runs commands, returns messages to send"""
         command = message.content.split()[0]
         responses = ReturnMessage()
         try:
@@ -107,7 +107,7 @@ class EchoBox:
         return responses
 
     def set_connection(self, message: Message) -> ReturnMessage:
-        """ Sets echo connection to given channel ID if found """
+        """Sets echo connection to given channel ID if found"""
         try:
             channel_id = int(message.content.split()[1])
         except ValueError:
@@ -128,7 +128,7 @@ class EchoBox:
         )
 
     def send_echo(self, message: Message) -> ReturnMessage:
-        """ Ensure we have a target channel that isn't expired """
+        """Ensure we have a target channel that isn't expired"""
         if not self.target_channel:
             return ReturnMessage(
                 "", "Use `echo!set [channel ID]` to set a target channel first"
@@ -146,14 +146,14 @@ class EchoBox:
             f"Message echo'ed to: {self.target_channel.name}",
         )
 
-    def stop_echo(self, _) -> ReturnMessage:
-        """ Removes target channel """
+    def stop_echo(self, _: Message) -> ReturnMessage:
+        """Removes target channel"""
         self.target_channel = None
         self.target_use_mark = 0.0
         return ReturnMessage("", "Echo target cleared.")
 
     async def on_message(self, msg: Message) -> None:
-        """ ON MESSAGE event hook """
+        """ON MESSAGE event hook"""
         if str(msg.channel.type) != "private" or not self.owner_id:
             return
         self.logger.debug("Got message: %s", msg)

--- a/modules/module_shoulderbirdparser.py
+++ b/modules/module_shoulderbirdparser.py
@@ -32,24 +32,24 @@ AUTO_LOAD: str = "ShoulderBirdParser"
 
 
 class ShoulderBirdParser:
-    """ Point of entry object for ShoulderBird module """
+    """Point of entry object for ShoulderBird module"""
 
     logger = logging.getLogger(__name__)
 
     def __init__(self, config_file: str = DEFAULT_CONFIG) -> None:
-        """ Loads config """
+        """Loads config"""
         self.__config = ShoulderBirdConfig(config_file)
         self.cli = ShoulderbirdCLI(self.__config)
 
-    def close(self):
-        """ Saves config state, breaks all references """
+    def close(self) -> None:
+        """Saves config state, breaks all references"""
         self.__config.save_config()
         del self.__config
 
     def get_matches(
         self, guild_id: str, user_id: str, clean_message: str
     ) -> List[BirdMember]:
-        """ Returns a list of BirdMembers whos searches match clean_message """
+        """Returns a list of BirdMembers whos searches match clean_message"""
         self.logger.debug(
             "get_matches: '%s', '%s', '%s'", guild_id, user_id, clean_message
         )
@@ -66,7 +66,7 @@ class ShoulderBirdParser:
 
     @classmethod
     def __is_valid_message(cls, message: Message) -> bool:
-        """ Tests for valid message to process """
+        """Tests for valid message to process"""
         if not isinstance(message, Message):
             cls.logger.error("Unknown arg type: %s", type(message))
             return False
@@ -82,7 +82,7 @@ class ShoulderBirdParser:
         return True
 
     async def on_message(self, message: Message) -> None:
-        """ Hook for discord client, async coro """
+        """Hook for discord client, async coro"""
         if not ShoulderBirdParser.__is_valid_message(message):
             return None
 
@@ -118,7 +118,7 @@ class ShoulderBirdParser:
     async def __send_match_dm(
         self, member: BirdMember, message: Message, guild: Guild
     ) -> None:
-        """ Private - send DM message to match. To be replaced with actions queue """
+        """Private - send DM message to match. To be replaced with actions queue"""
         try:
             target: Member = guild.get_member(int(member.member_id))
         except ValueError:
@@ -134,7 +134,7 @@ class ShoulderBirdParser:
 
     @staticmethod
     async def __send_dm(target: Member, content: str) -> None:
-        """ Private, sends a DM to target """
+        """Private, sends a DM to target"""
         if target.dm_channel is None:
             await target.create_dm()
         if target.dm_channel:

--- a/modules/shoulderbirdconfig.py
+++ b/modules/shoulderbirdconfig.py
@@ -13,6 +13,8 @@ Discord : Preocts#8196
 Git Repo: https://github.com/Preocts/Egg_Bot
 """
 import logging
+from typing import Any
+from typing import Dict
 from typing import List
 from typing import Set
 
@@ -24,19 +26,19 @@ DEFAULT_CONFIG = "configs/shoulderbird.json"
 
 
 class BirdMember:
-    """ Model class for member level config values """
+    """Model class for member level config values"""
 
     # pylint: disable=too-few-public-methods
 
-    def __init__(self, guild_id: str, member_id: str, **kwargs) -> None:
+    def __init__(self, guild_id: str, member_id: str, **kwargs: Any) -> None:
         self.guild_id = guild_id
         self.member_id = member_id
         self.regex: str = kwargs.get("regex", "")
         self.toggle: bool = kwargs.get("toggle", True)
         self.ignore: Set[str] = set(kwargs.get("ignore", []))
 
-    def to_dict(self) -> dict:
-        """ Converts values to dict for use in JSON """
+    def to_dict(self) -> Dict[str, Any]:
+        """Converts values to dict for use in JSON"""
         return {
             "guild_id": self.guild_id,
             "member_id": self.member_id,
@@ -47,12 +49,12 @@ class BirdMember:
 
 
 class ShoulderBirdConfig:
-    """ Shoulder Bird Config class, CRUD config operations """
+    """Shoulder Bird Config class, CRUD config operations"""
 
     logger = logging.getLogger(__name__)
 
     def __init__(self, config_file: str = DEFAULT_CONFIG) -> None:
-        """ Init and load config """
+        """Init and load config"""
         self.logger.info("Initializing Shoulder Bird Parser")
         self.__configclient = ConfigFile()
         self.__configclient.load(config_file)
@@ -60,29 +62,29 @@ class ShoulderBirdConfig:
             self.__configclient.create("module", MODULE_NAME)
             self.__configclient.create("version", MODULE_VERSION)
 
-    def __load_guild(self, guild_id: str) -> dict:
-        """ Load a specific guild from config. Will create guild if not found """
+    def __load_guild(self, guild_id: str) -> Dict[str, Any]:
+        """Load a specific guild from config. Will create guild if not found"""
         self.logger.debug("load_guild: '%s'", guild_id)
         if guild_id not in self.__configclient.config:
             self.__configclient.create(guild_id, {})
         return self.__configclient.read(guild_id)
 
     def __save_member_to_guild(self, guild_id: str, member: BirdMember) -> None:
-        """ Save a specific member to a guild. Creates new or overwrites existing """
+        """Save a specific member to a guild. Creates new or overwrites existing"""
         guild_config = self.__load_guild(guild_id)
         guild_config[member.member_id] = member.to_dict()
         self.__configclient.update(guild_id, guild_config)
 
     def reload_config(self) -> bool:
-        """ Reloads current config file without saving """
+        """Reloads current config file without saving"""
         return self.__configclient.load()
 
     def save_config(self) -> bool:
-        """ Saves current config to file """
+        """Saves current config to file"""
         return self.__configclient.save()
 
     def member_list_all(self, member_id: str) -> List[BirdMember]:
-        """ Returns all configs for member across guilds, can return empty list """
+        """Returns all configs for member across guilds, can return empty list"""
         self.logger.debug("member_list_all: '%s'", member_id)
         config_list = []
         for guild in self.__configclient.config.values():
@@ -91,7 +93,7 @@ class ShoulderBirdConfig:
         return config_list
 
     def guild_list_all(self, guild_id: str) -> List[BirdMember]:
-        """ Returns all configs within a single guild, can return empty list """
+        """Returns all configs within a single guild, can return empty list"""
         self.logger.debug("guild_list_all: '%s'", guild_id)
         config_list = []
         for member in self.__load_guild(guild_id).values():
@@ -99,12 +101,12 @@ class ShoulderBirdConfig:
         return config_list
 
     def load_member(self, guild_id: str, member_id: str) -> BirdMember:
-        """ Load a member from a guild. Will return empty member if not found """
+        """Load a member from a guild. Will return empty member if not found"""
         self.logger.debug("load_member: '%s', '%s'", guild_id, member_id)
         member = self.__load_guild(guild_id).get(member_id)
         return BirdMember(**member) if member else BirdMember(guild_id, member_id)
 
-    def save_member(self, guild_id: str, member_id: str, **kwargs) -> BirdMember:
+    def save_member(self, guild_id: str, member_id: str, **kwargs: Any) -> BirdMember:
         """Save (creating or updating) a member to a guild
 
         Keyword Args:
@@ -121,7 +123,7 @@ class ShoulderBirdConfig:
         return member_config
 
     def delete_member(self, guild_id: str, member_id: str) -> bool:
-        """ Deletes member from specific guild, returns false if not found """
+        """Deletes member from specific guild, returns false if not found"""
         self.logger.debug("delete_member: '%s', '%s'", guild_id, member_id)
         guild_config = self.__load_guild(guild_id)
         deleted_value = guild_config.pop(member_id, None)

--- a/mypy.ini
+++ b/mypy.ini
@@ -1,11 +1,11 @@
 # mypy globals
 
 [mypy]
-; check_untyped_defs = true
-; disallow_any_generics = true
-; disallow_incomplete_defs = true
-; disallow_untyped_defs = true
-; no_implicit_optional = true
+check_untyped_defs = true
+disallow_any_generics = true
+disallow_incomplete_defs = true
+disallow_untyped_defs = true
+no_implicit_optional = true
 
 [mypy-tests.*]
 disallow_untyped_defs = false

--- a/tests/test_configfile.py
+++ b/tests/test_configfile.py
@@ -7,16 +7,16 @@ from eggbot.configfile import ConfigFile
 
 
 class TestCoreConfig(unittest.TestCase):
-    """ Test suite """
+    """Test suite"""
 
     def test_properties(self):
-        """ Unit Test """
+        """Unit Test"""
         config = ConfigFile("./tests/fixtures/mock_config.json")
         config.load()
         self.assertIsInstance(config.config, dict)
 
     def test_load(self):
-        """ Unit Test """
+        """Unit Test"""
         config = ConfigFile("./tests/fixtures/mock_config.json")
         # Missing file
         config.load("invalid.file")
@@ -31,14 +31,14 @@ class TestCoreConfig(unittest.TestCase):
         self.assertFalse(config.config)
 
     def test_config_crud(self):
-        """ Unit Test """
+        """Unit Test"""
         random.seed()
         key = f"unitTest{random.randint(1000,10000)}"  # nosec
         config = ConfigFile("./tests/fixtures/mock_config.json")
 
         self.assertTrue(config.create(key, "Test Value"))
         self.assertIn(key, config.config.keys())
-        self.assertFalse(config.create(12345, "Test Value"))
+        self.assertFalse(config.create(12345, "Test Value"))  # type: ignore
         self.assertNotIn(12345, config.config.keys())
         self.assertFalse(config.create(key, "Test Value"))
 
@@ -55,7 +55,7 @@ class TestCoreConfig(unittest.TestCase):
         self.assertFalse(config.delete(key))
 
     def test_save(self):
-        """ Unit Test """
+        """Unit Test"""
         random.seed()
         key = f"unitTest{random.randint(1000,10000)}"  # nosec
 
@@ -75,7 +75,7 @@ class TestCoreConfig(unittest.TestCase):
         self.assertTrue(config.config)
 
     def test_unload(self):
-        """ Empty current config, reload from same file """
+        """Empty current config, reload from same file"""
         config = ConfigFile("./tests/fixtures/mock_config.json")
         config.load()
         self.assertTrue(config.config)

--- a/tests/test_discordclient.py
+++ b/tests/test_discordclient.py
@@ -8,18 +8,18 @@ from eggbot.discordclient import DiscordClient
 
 
 class TestDiscordClient(unittest.TestCase):
-    """ Discord Client Abstract """
+    """Discord Client Abstract"""
 
     def test_singleton_assignment(self):
-        """ One client and only one client """
+        """One client and only one client"""
         client1 = DiscordClient()
         client2 = DiscordClient()
         self.assertIs(client1, client2)
 
     def test_assert_run_with_secret(self):
-        """ Use all the things """
+        """Use all the things"""
         client = DiscordClient()
         client.set_secret("Hello there")
         with patch.object(client.client, "run") as mock_discord:
-            self.assertIsNone(client.run())
+            client.run()
             mock_discord.assert_called_once_with("Hello there")

--- a/tests/test_eggbotcore.py
+++ b/tests/test_eggbotcore.py
@@ -11,17 +11,17 @@ from eggbot.models.eventtype import EventType
 
 
 def run_loop(func, *args, **kwargs):
-    """ Hack to test async functions """
+    """Hack to test async functions"""
     loop = asyncio.get_event_loop()
     result = loop.run_until_complete(func(*args, **kwargs))
     return result
 
 
 class TestEggbotCore(unittest.TestCase):
-    """ Testing suite """
+    """Testing suite"""
 
     def test_attributes(self):
-        """ These need to exist """
+        """These need to exist"""
         eggbot = EggBotCore()
         self.assertIsInstance(eggbot.discord_, ec.DiscordClient)
         self.assertIsInstance(eggbot.event_subs, ec.EventSubs)
@@ -29,17 +29,17 @@ class TestEggbotCore(unittest.TestCase):
         self.assertIsInstance(eggbot.env_vars, ec.LoadEnv)
 
     def test_config_load(self):
-        """ Pass fail on config load """
+        """Pass fail on config load"""
         eggbot = EggBotCore()
         self.assertTrue(eggbot.load_config())
         with patch.object(eggbot.core_config, "load", return_value=False):
             self.assertFalse(eggbot.load_config())
-        eggbot.core_config = None
+        eggbot.core_config = None  # type: ignore
         with self.assertRaises(Exception):
             eggbot.load_config()
 
     def test_launch_bot(self):
-        """ Should exit clean after doing things """
+        """Should exit clean after doing things"""
         eggbot = EggBotCore()
         with patch.object(eggbot.discord_, "run") as mock_client:
             self.assertEqual(eggbot.launch_bot(), 0)
@@ -49,12 +49,12 @@ class TestEggbotCore(unittest.TestCase):
             with self.assertRaises(Exception):
                 eggbot.launch_bot()
 
-        eggbot.env_vars = None
+        eggbot.env_vars = None  # type: ignore
         with self.assertRaises(Exception):
             eggbot.launch_bot()
 
     def test_join_ignore_me(self):
-        """ On Joins should ignore bot actions """
+        """On Joins should ignore bot actions"""
         eggbot = EggBotCore()
         with patch.object(eggbot.discord_, "client") as mock_client:
             mock_client.user.id = "123456789"
@@ -75,7 +75,7 @@ class TestEggbotCore(unittest.TestCase):
             self.assertFalse(run_loop(eggbot.on_member_join, mock_member))
 
     def test_message_ignore_me(self):
-        """ Message events should ignore bot chatter """
+        """Message events should ignore bot chatter"""
         eggbot = EggBotCore()
         with patch.object(eggbot.discord_, "client") as mock_client:
             mock_client.user.id = "123456789"
@@ -96,7 +96,7 @@ class TestEggbotCore(unittest.TestCase):
             self.assertFalse(run_loop(eggbot.on_message, mock_message))
 
     def test_join_pub_events(self):
-        """ Make sure join calls the "on_join" sub list """
+        """Make sure join calls the "on_join" sub list"""
         eggbot = EggBotCore()
         _join_sub_01 = unittest.mock.AsyncMock()
         _join_sub_02 = unittest.mock.AsyncMock()
@@ -115,7 +115,7 @@ class TestEggbotCore(unittest.TestCase):
         _join_sub_02.assert_called_with(mock_member)
 
     def test_message_pub_events(self):
-        """ Make sure messages call the "on_message" sub list """
+        """Make sure messages call the "on_message" sub list"""
         eggbot = EggBotCore()
         message_sub_01 = unittest.mock.AsyncMock()
         message_sub_02 = unittest.mock.AsyncMock()

--- a/tests/test_eventsubs.py
+++ b/tests/test_eventsubs.py
@@ -12,16 +12,16 @@ from eggbot.models.eventtype import EventType
 
 
 class TestEventSub(unittest.TestCase):
-    """ Test Suite """
+    """Test Suite"""
 
     def test_create_and_get_events(self):
-        """ Manual add and get """
+        """Manual add and get"""
         events_client = EventSubs()
         mockclass = MockClass()
 
-        events_client.add(EventType.ON_MEMBER_JOIN, mockclass.method01)
-        events_client.add(EventType.ON_DISCONNECT, mockclass.method02)
-        events_client.add(EventType.ON_MESSAGE, mockfunc)
+        events_client.add(EventType.ON_MEMBER_JOIN, mockclass.method01)  # type: ignore
+        events_client.add(EventType.ON_DISCONNECT, mockclass.method02)  # type: ignore
+        events_client.add(EventType.ON_MESSAGE, mockfunc)  # type: ignore
 
         self.assertEqual(
             type(events_client.get(EventType.ON_MEMBER_JOIN)[0]),
@@ -37,22 +37,20 @@ class TestEventSub(unittest.TestCase):
 
 
 class MockClass:
-    """ Mock class for mock test """
+    """Mock class for mock test"""
 
     def __init__(self) -> None:
-        """ Mock """
+        """Mock"""
         self.ran = False
 
     def method01(self) -> None:
-        """ Mock """
+        """Mock"""
         self.ran = True
 
-    def method02(self, arg1: str, arg2: dict) -> bool:
-        """ Mock """
+    def method02(self) -> None:
+        """Mock"""
         self.ran = True
-        return arg1 in arg2
 
 
-def mockfunc(arg1: str, arg2: dict) -> bool:
-    """ Mock """
-    return arg1 not in arg2
+def mockfunc() -> None:
+    """Mock"""

--- a/tests/test_module_chatkudos.py
+++ b/tests/test_module_chatkudos.py
@@ -27,8 +27,8 @@ from modules.module_chatkudos import MODULE_VERSION
 
 
 @pytest.fixture(scope="function", name="kudos")
-def fixture_kudos() -> Generator:
-    """ Fixture """
+def fixture_kudos() -> Generator[ChatKudos, None, None]:
+    """Fixture"""
     kudos = ChatKudos("./tests/fixtures/mock_chatkudos.json")
     # disable writing to the fixture file
     with patch.object(kudos.config, "save"):
@@ -37,7 +37,7 @@ def fixture_kudos() -> Generator:
 
 @pytest.fixture(scope="function", name="message")
 def fixture_message() -> Mock:
-    """ Fixture """
+    """Fixture"""
     members: List[Mock] = [
         Mock(display_name="Tester01"),
         Mock(display_name="Tester02"),
@@ -64,7 +64,7 @@ def fixture_message() -> Mock:
 
 @pytest.fixture(scope="function", name="async_message")
 def fixture_async_message() -> AsyncMock:
-    """ Fixture """
+    """Fixture"""
     message = AsyncMock()
     message.content = "<#!111> + <!222#> + Oh yeah baby!"
     message.guild.name = "Testing Guild"
@@ -82,19 +82,19 @@ def fixture_async_message() -> AsyncMock:
 
 
 def test_does_fixture_need_updating(kudos: ChatKudos) -> None:
-    """ Confirm version and module names match, sanity check """
+    """Confirm version and module names match, sanity check"""
     assert kudos.config.read("module") == MODULE_NAME
     assert kudos.config.read("version") == MODULE_VERSION
 
 
 def test_command_config(kudos: ChatKudos) -> None:
-    """ Ensure all commands have attr that exist """
+    """Ensure all commands have attr that exist"""
     for attr in COMMAND_CONFIG.values():
         assert getattr(kudos, attr, None) is not None, f"Missing {attr} attribute"
 
 
 def test_load_guild_found(kudos: ChatKudos) -> None:
-    """ Load a guild that exists, check values """
+    """Load a guild that exists, check values"""
     result = kudos.get_guild("111")
     assert result.max == -1
     assert result.users
@@ -102,14 +102,14 @@ def test_load_guild_found(kudos: ChatKudos) -> None:
 
 
 def test_load_guild_not_found(kudos: ChatKudos) -> None:
-    """ Load a guild that doesn't exist, check defaults """
+    """Load a guild that doesn't exist, check defaults"""
     result = kudos.get_guild("999")
     assert result.max == 5
     assert not result.users
 
 
 def test_save_guild_exists(kudos: ChatKudos) -> None:
-    """ Save explict changes, confirm existing don't change """
+    """Save explict changes, confirm existing don't change"""
     kudos.save_guild("111", max=-1, gain_message="TEST")
     kudos.save_guild("111", loss_message="TEST02")
 
@@ -119,7 +119,7 @@ def test_save_guild_exists(kudos: ChatKudos) -> None:
 
 
 def test_save_guild_not_exists(kudos: ChatKudos) -> None:
-    """ Save changes to a guild that does't exist """
+    """Save changes to a guild that does't exist"""
     kudos.save_guild("999", max=-1, gain_message="TEST")
     kudos.save_guild("999", loss_message="TEST02")
 
@@ -129,7 +129,7 @@ def test_save_guild_not_exists(kudos: ChatKudos) -> None:
 
 
 def test_adjust_max(kudos: ChatKudos, message: Mock) -> None:
-    """ Change max for existing and non-existing guild """
+    """Change max for existing and non-existing guild"""
     message.content = "kudos!max 10"
     result = kudos.parse_command(message)
     assert "Max points now: 10" in result
@@ -144,14 +144,14 @@ def test_adjust_max(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_adjust_max_invalid(kudos: ChatKudos, message: Mock) -> None:
-    """ Provide an invalid format to the command """
+    """Provide an invalid format to the command"""
     message.content = "kudos!max 10 points"
     result = kudos.parse_command(message)
     assert "Usage:" in result
 
 
 def test_adjust_gain_messages(kudos: ChatKudos, message: Mock) -> None:
-    """ Adjust the gain messages on existing and non-existing guilds """
+    """Adjust the gain messages on existing and non-existing guilds"""
     message.content = "kudos!gain This is gain"
     result = kudos.parse_command(message)
     assert "Message has been set." in result
@@ -164,7 +164,7 @@ def test_adjust_gain_messages(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_adjust_loss_messages(kudos: ChatKudos, message: Mock) -> None:
-    """ Adjust the loss messages on existing and non-existing guilds """
+    """Adjust the loss messages on existing and non-existing guilds"""
     message.content = "kudos!loss This is loss"
     result = kudos.parse_command(message)
     assert "Message has been set." in result
@@ -177,14 +177,14 @@ def test_adjust_loss_messages(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_adjust_message_empty(kudos: ChatKudos, message: Mock) -> None:
-    """ Confirm an empty message is skipped """
+    """Confirm an empty message is skipped"""
     message.content = "kudos!gain"
     result = kudos.parse_command(message)
     assert not result
 
 
 def test_add_remove_user_list(kudos: ChatKudos, message: Mock) -> None:
-    """ Add, then remove, a user from user list """
+    """Add, then remove, a user from user list"""
     message.role_mentions = []  # Clear to focus test scope
     message.content = "kudos!user @Tester"  # mention is provided by fixture
 
@@ -198,7 +198,7 @@ def test_add_remove_user_list(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_add_remove_role_list(kudos: ChatKudos, message: Mock) -> None:
-    """ Add, then remove, a role from role list """
+    """Add, then remove, a role from role list"""
     message.mentions = []  # Clear to focus test scope
     message.content = "kudos!role @Cool Kats"  # role_mention provided by fixture
 
@@ -212,7 +212,7 @@ def test_add_remove_role_list(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_add_remove_list_empty(kudos: ChatKudos, message: Mock) -> None:
-    """ Confirm message with no mentions is skipped """
+    """Confirm message with no mentions is skipped"""
     message.role_mentions = []
     message.mentions = []
     message.content = "kudos!user This is just an empty message"
@@ -222,7 +222,7 @@ def test_add_remove_list_empty(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_lock_toggle(kudos: ChatKudos, message: Mock) -> None:
-    """ Turn the lock on and off again, fixture starts locked """
+    """Turn the lock on and off again, fixture starts locked"""
     message.content = "kudos!lock"
 
     result = kudos.parse_command(message)
@@ -235,7 +235,7 @@ def test_lock_toggle(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_help(kudos: ChatKudos, message: Mock) -> None:
-    """ Does this have a help? """
+    """Does this have a help?"""
     message.content = "kudos!help"
 
     result = kudos.parse_command(message)
@@ -243,7 +243,7 @@ def test_help(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_board_all(kudos: ChatKudos, message: Mock) -> None:
-    """ Print the board without # limit """
+    """Print the board without # limit"""
     message.content = "kudos!board Give me all the board"
 
     result = kudos.parse_command(message)
@@ -253,7 +253,7 @@ def test_board_all(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_board_limited(kudos: ChatKudos, message: Mock) -> None:
-    """ Print the board with a # limit """
+    """Print the board with a # limit"""
     message.content = "kudos!board 1"
 
     result = kudos.parse_command(message)
@@ -263,7 +263,7 @@ def test_board_limited(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_find_kudos(kudos: ChatKudos, message: Mock) -> None:
-    """ Return the accurate Kudos count for messages """
+    """Return the accurate Kudos count for messages"""
     message.content = "<#!111> ++-++-++ <!222#> just kidding"
     message.mentions = [
         Mock(id="111", display_name="Tester01"),
@@ -282,7 +282,7 @@ def test_find_kudos(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_format_message() -> None:
-    """ Message formatting """
+    """Message formatting"""
     msg = "Fantastic [NAME], have [POINTS] Kudos points. You now have [TOTAL] in total."
     expect = "Fantastic Tester01, have 15 Kudos points. You now have 30 in total."
     kudos = Kudos("111", "Tester01", 15, 15)
@@ -294,7 +294,7 @@ def test_format_message() -> None:
 
 @pytest.mark.asyncio
 async def test_onmessage_kudos(kudos: ChatKudos, async_message: AsyncMock) -> None:
-    """ Give two Kudos. Config should update """
+    """Give two Kudos. Config should update"""
     await kudos.on_message(async_message)
 
     scores = kudos.get_guild("111").scores
@@ -307,7 +307,7 @@ async def test_onmessage_kudos(kudos: ChatKudos, async_message: AsyncMock) -> No
 
 @pytest.mark.asyncio
 async def test_onmessage_command(kudos: ChatKudos, async_message: AsyncMock) -> None:
-    """ Give a command, ensure we hit the command path """
+    """Give a command, ensure we hit the command path"""
     async_message.content = "kudos!help"
 
     await kudos.on_message(async_message)
@@ -316,7 +316,7 @@ async def test_onmessage_command(kudos: ChatKudos, async_message: AsyncMock) -> 
 
 
 def test_is_command_allowed(kudos: ChatKudos, message: Mock) -> None:
-    """ Case checks for accessing commands """
+    """Case checks for accessing commands"""
     kudos.save_guild("111", lock=True)
     # Server Owner
     message.content = "kudos!help"
@@ -334,7 +334,7 @@ def test_is_command_allowed(kudos: ChatKudos, message: Mock) -> None:
 
 
 def test_is_kudos_allowed(kudos: ChatKudos, message: Mock) -> None:
-    """ Case checks for accessing Kudos, only new test is role list """
+    """Case checks for accessing Kudos, only new test is role list"""
 
     # On Role List
     Roles = namedtuple("Roles", ["id"])

--- a/tests/test_module_shoulderbirdparser.py
+++ b/tests/test_module_shoulderbirdparser.py
@@ -17,28 +17,28 @@ from modules.module_shoulderbirdparser import ShoulderBirdParser
 
 @pytest.fixture(scope="function", name="parser")
 def fixture_parser() -> ShoulderBirdParser:
-    """ fixture """
+    """fixture"""
     return ShoulderBirdParser("./tests/fixtures/mock_shoulderbirdparser.json")
 
 
 def test_positive_match_simple(parser: ShoulderBirdParser) -> None:
-    """ Positive matches, also tests toggle """
+    """Positive matches, also tests toggle"""
     matches = parser.get_matches("101", "Delta", "This is a test")
     assert len(matches) == 2
     for idx, match in enumerate(matches):
         assert match.member_id == f"10{idx + 1}"
 
 
-def test_positive_match_complex(parser: ShoulderBirdParser):
-    """ Positive match, tests for case agnostic """
+def test_positive_match_complex(parser: ShoulderBirdParser) -> None:
+    """Positive match, tests for case agnostic"""
     msg = "I should hope this TESTsuite passes!"
     matches = parser.get_matches("102", "Delta", msg)
     assert len(matches) == 1
     assert matches[0].member_id == "101"
 
 
-def test_word_bound_regex_complex_search(parser: ShoulderBirdParser):
-    """ Negitive match, ensure word bound is applied """
+def test_word_bound_regex_complex_search(parser: ShoulderBirdParser) -> None:
+    """Negitive match, ensure word bound is applied"""
     msg = "appreciated"
     matches = parser.get_matches("101", "Delta", msg)
     assert len(matches) == 0

